providers/implementations/ciphers/cipher_aes_siv.c

```diff
@@ -185,7 +185,6 @@ static int aes_siv_get_ctx_params(void *vctx, OSSL_PARAM params[])
 static const OSSL_PARAM aes_siv_known_gettable_ctx_params[] = {
     OSSL_PARAM_size_t(OSSL_CIPHER_PARAM_KEYLEN, NULL),
     OSSL_PARAM_size_t(OSSL_CIPHER_PARAM_AEAD_TAGLEN, NULL),
-    OSSL_PARAM_uint(OSSL_CIPHER_PARAM_SPEED, NULL),
```

Comment:
This is noted as supported by SIV in the documentation, why remove it ?

Reply:
Becuase this has only ever been a settable control. And it is not implemented by the get_ctx_params() (And doesnt need to be).